src/sentry/models/groupresolution.py

```diff
@@ -43,3 +45,36 @@ class Meta:
         app_label = 'sentry'
 
     __repr__ = sane_repr('group_id', 'release_id')
+
+    @classmethod
+    def has_resolution(cls, group, release):
+        """
+        Determine if a resolution exists for the given group and release.
+
+        This is used to suggest if a regression has occurred.
+        """
+        try:
+            res_type, res_release, res_release_datetime = cls.objects.filter(
```

Comment:
Would you want to sort this query on `datetime` or something? It seems error prone if there are multiple GroupResolution rows for the same group, across different releases. We're simply grabbing the first one in an arbitrary order. Should we not also filter on `release=release` ?